app/views/questionnaire.py

```diff
@@ -306,7 +308,8 @@ def _render_page(full_routing_path, block, current_location, post_form=None):
 
 
 def _is_valid_location(routing_path, location):
-    return location in routing_path
+    is_survey_complete = not completeness.get_first_incomplete_location_in_survey()
+    return location in routing_path or is_survey_complete
```

Comment:
Im not sure how this helps?

Reply:
I did that for a reason... but I can't remember what it was now. I'll find out and clarify.